README.md

```diff
@@ -35,7 +35,18 @@ For more details, see [the quick start](http://gigalixir.readthedocs.io/en/lates
    2. `gigalixir login`
    3. `APP_NAME=$(gigalixir create --name course-planner)`
    4. `gigalixig set_config $APP_NAME DATABASE_URL $YOUR_DB_URL`
+      
+      If you don't have a database, you can create one by running `gigalixir create_database $APP_NAME` and waiting for it to become `AVAILABLE`.
    5. `git push gigalixir master`
-   6. `gigalixir migrate $APP_NAME`
-
-
+   
+   6. `MIX_ENV=prod mix release --env=prod`
```

Comment:
why is `MIX_ENV=prod mix release --env=prod` necessary? it doesn't seem like the local release is used later

Reply:
I'm not sure, it's mentioned in gigalixir's workaround step.
Maybe it checks if the release is being built correctly?